README.md

```diff
@@ -446,6 +468,15 @@ Returns an array with the first `n` entries omitted.
 
 **[⬆️ back to top](#available-helpers)**
 
+#### `reduce`
+Reduce an array to a value.
+
+```hbs
+{{reduce (action 'sum') (array 1 2 3) 0}}
+```
+
+The last argument is initial value. If you omit it, undefined will be used.
```

Comment:
I think it should be the second argument: `{{reduce (action "sum") 0 (array 1 2 3)}}`